src/Actions/actions.js

```diff
@@ -12,7 +12,21 @@ function changeDisplayStyle(style) {
     }
 }
 
+function shouldDisplayModal(){
+    return {
+        type: 'DISPLAY_MODAL',
+    }
+}
+
+function hideModal() {
+    return {
+        type: 'HIDE_MODAL',
```

Comment:
Do you need two different action types?

Can you just have a `TOGGLE_MODAL` and the data tells whether it's on or off?